addon/components/file-uploader/component.js

```diff
@@ -14,7 +14,7 @@ export default Component.extend({
   method: 'PUT',
   attribute: 'file',
 
-  text: 'Add an file',
+  text: 'Add a file',
```

Comment:
Is 'Upload File' more precise ?

Reply:
each application can reset this text actually, but yeah I can put Upload File by default

Reply:
in Publishr campaigns' setup for example, it's called "Select File"